Makefile

```diff
@@ -51,19 +53,22 @@ install:
 		" \
 		$(BUILD_PATH)
 
-ci-build: build/operator-sdk-$(VERSION)-x86_64-linux-gnu ci-install
+ci-build: build/operator-sdk-$(VERSION) ci-install
 
 ci-install:
-	mv build/operator-sdk-$(VERSION)-x86_64-linux-gnu build/operator-sdk
+	mv build/operator-sdk-$(VERSION) build/operator-sdk
 
-release_x86_64 := \
+release_builds := \
 	build/operator-sdk-$(VERSION)-x86_64-linux-gnu \
-	build/operator-sdk-$(VERSION)-x86_64-apple-darwin
+	build/operator-sdk-$(VERSION)-x86_64-apple-darwin \
+	build/operator-sdk-$(VERSION)-ppc64le-linux-gnu
 
-release: clean $(release_x86_64) $(release_x86_64:=.asc)
+release: clean $(release_builds) $(release_builds:=.asc)
```

Comment:
If we're supporting ppc64le release binaries now then we should update the release doc to mention that.
https://github.com/operator-framework/operator-sdk/blob/master/doc/dev/release.md#operating-systems-and-architectures

Reply:
ok for a followon pr so it dosn't invalidate this lgtm again?

Reply:
Let's go ahead and fix it here. I can babysit the builds if necessary :)

Reply:
Yep we can do a follow up. Just a nit.